src/playbacks/no_op/no_op.js

```diff
@@ -14,13 +14,24 @@ export default class NoOp extends Playback {
     return {'data-no-op': ''}
   }
 
+  getNoOpMessage(){
+    var messages = {}
+    messages['en'] = "Your browser does not support the playback of this video. Try to use a different browser."
+    messages['es'] = "Su navegador no soporta la reproducción de un video. Trate de usar un navegador diferente."
+    messages['pt'] = "Seu navegador não supporta a reprodução deste video. Tente usar um outro navegador."
+    messages['en-US'] = messages['en']
+    messages['es-419'] = messages['es']
+    messages['pt-BR'] = messages['pt']
+    return messages[window.navigator.language]
```

Comment:
nice catch @leandromoreira I created the installNavigator function inside the NoOP I think this should be fine.